util/automation.py

```diff
@@ -32,8 +32,8 @@
 
 DEEPSPEECH_CLONE_PATH  = os.path.abspath(os.environ.get('ds_clone_path', './ds_exec_clone/'))
 
-CACHE_DIR = os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer')
-DATA_DIR  = os.path.join(BaseDirectory.xdg_data_home, 'deepspeech_wer')
+CACHE_DIR = os.path.abspath(os.path.join(BaseDirectory.xdg_cache_home, 'deepspeech_wer'))
+DATA_DIR  = os.path.abspath(os.path.join(os.environ.get('ds_dataroot', BaseDirectory.xdg_data_home), 'deepspeech_wer'))
```

Comment:
Pick a value set from env for `ds_dataroot` and defaulting to XDG_DATA_HOME if none is set.

When this lands, one should ensure that sever-side the content of ~/.local/share/deepspeech_wer/ is being moved to /data